[2021-06-25 09:19:36,996] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2020-03-13T00:00:00+00:00 [queued]>
[2021-06-25 09:19:37,299] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2020-03-13T00:00:00+00:00 [queued]>
[2021-06-25 09:19:37,300] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:19:37,300] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-25 09:19:37,301] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:19:37,436] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2020-03-13T00:00:00+00:00
[2021-06-25 09:19:37,462] {standard_task_runner.py:52} INFO - Started process 8567 to run task
[2021-06-25 09:19:37,588] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2020-03-13T00:00:00+00:00', '--job-id', '316', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp_quzdmy1', '--error-file', '/tmp/tmpk0rdziif']
[2021-06-25 09:19:37,589] {standard_task_runner.py:77} INFO - Job 316: Subtask task_4
[2021-06-25 09:19:38,223] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2020-03-13T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:19:39,042] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2020-03-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-13T00:00:00+00:00
[2021-06-25 09:19:39,098] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:19:39,139] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:19:39,299] {subprocess.py:75} INFO - Output:
[2021-06-25 09:19:42,368] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:19:43,147] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20200313T000000, start_date=20210625T091937, end_date=20210625T091943
[2021-06-25 09:19:43,403] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:19:43,475] {local_task_job.py:151} INFO - Task exited with return code 0
